src/sparseml/core/session.py

```diff
@@ -146,6 +148,8 @@ def initialize(
         start: Optional[float] = None,
         steps_per_epoch: Optional[int] = None,
         batches_per_step: Optional[int] = None,
+        loggers: Union[None, LoggerManager, List[BaseLogger]] = None,
+        model_log_cadence: Optional[float] = None,
```

Comment:
Don't think we need a separate model log and general loggers frequency. probably fine to include all together

Reply:
This allows a easy knob for users to control `model_log_cadence`
especially for cases where users don't specify a `LoggerManager` during `session.initialize`

Without `model_log_cadence` the way to set cadence is as follows:
```python
session.initialize(..., 
loggers=LoggerManager(loggers=[PythonLogger()], log_frequency= 0.5))
```
Which is not very intuitive and users need to know how to initialize the `LoggerManager`

Using `model_log_cadence` the UX is simpler:
```python
# users can also avoid specify loggers in session.initialize
session.initialize(..., model_log_cadence=1) 
```

If we bake `model_log_cadence` into `LoggerManager` then modifier level logs would have to specify correct `LoggerManager.log_frequency` before calling `LoggerManager.log_ready()`


Can still update if not convinced